CoreInteractiveGraph/src/au/gov/asd/tac/constellation/graph/interaction/visual/DefaultInteractionEventHandler.java

```diff
@@ -416,7 +416,7 @@ public void mouseDragged(final MouseEvent event) {
                             if (zAxisRotation) {
                                 CameraUtilities.spin(camera, visualInteraction.convertTranslationToSpin(from, to));
                             } else {
-                                Vector2i yawPitch = visualInteraction.adjustPitchYawCoords(from.x - to.x, from.y - to.y);
+                                Vector2i yawPitch = new Vector2i(from.x - to.x, from.y - to.y);
```

Comment:
How come it isn't adjusted anymore here?

Reply:
We don't need to change any of the world coordinates, the only conversion is in clip space.